### PR TITLE
look for a .stylintrc file when formatting stylus code

### PR DESCRIPTION
**Pull request by ntraut**
For issue #2689.

### CODE REVIEW

File: server/src/modes/style/stylus/index.ts [hunk 1]
```diff
@@ -101,8 +104,22 @@ export function getStylusMode(
       const stylusSupremacyFormattingOptions = stylusSupremacy.createFormattingOptions(
         env.getConfig().stylusSupremacy || {}
       );
+
+      // read .stylintrc file if it exists and give it priority over VS Code settings
+      const stylintrcPath = path.join(env.getRootPathForConfig(), '.stylintrc');
```
Comment (yoyo930021) [inline server/src/modes/style/stylus/index.ts:109]: Maybe use `env.getProjectRoot()` ?

**Review by yoyo930021 — CHANGES_REQUESTED** [review]
LGTM, only a little problem.

File: server/src/modes/style/stylus/index.ts (hunk 1 above)
Comment (ntraut) [reply server/src/modes/style/stylus/index.ts]: Changed. `env.getProjectRoot()` gives me the same result as `env.getRootPathForConfig()`.

Comment (yoyo930021) [reply server/src/modes/style/stylus/index.ts]: They are sometimes different.

**Review by yoyo930021 — APPROVED** [review]

**Pull request merged**